ush/HWP_tools.py

```diff
@@ -25,6 +38,15 @@ def check_restart_files(hourly_hwpdir, fcst_dates):
     return(hwp_avail_hours, hwp_non_avail_hours)
 
 def copy_missing_restart(nwges_dir, hwp_non_avail_hours, hourly_hwpdir):
+    """Check the restart files.
+
+    Args:
+         nwges_dir: directory
+         hwp_non_avail_hours: ???
+         hourly_hwpdir: ???

```

Comment:
What do these params mean?

Reply:
@edwardhartnett I reached out to the RRFS smoke people with your questions.

Reply:
nwges_dir holds the boundary, INPUT, and RESTART files (e.g, described here: https://github.com/ufs-community/ufs-srweather-app/issues/654; The use of NWGES is a bit of a legacy variable name in production. It is often used to define a directory holding DA-relevant files (such as first guess and analysis files) outside of the main COM output directory. It could be changed, but not sure if that would negatively impact those in GSL/EMC that deal with the DA aspects of RRFS.)

Reply:
hwp_non_avail_hours is a list of hours without available RESTART files

Reply:
OK, great I have incorporated all that into the docs.

Now the only thing left are the values defined in the returned lists...